site/_components/item-grid.njk

```diff
@@ -17,7 +17,10 @@
  #}-->
 {% from 'external-community.njk' import externalCommunity %} {% from 'card-subnav.njk' import cardSubnav %} {% from 'icon-card.njk' import iconCard %} {% from 'card.njk' import card %} {% macro itemGrid(title, items, modifiers, locale, defaultText) %} {% if items.length > 0 %}
 <section class="item-grid {{modifiers.custom}} {{'item-grid--icon-card' if modifiers.variant == 'icon'}} {{ 'item-grid--subnav' if modifiers.variant == 'subnav' }} {{'item-grid--community' if modifiers.variant == 'community'}} {{'item-grid--circled' if modifiers.shapeOn }}">
-  <h3 class="item-grid__title type--h3">{{title}}</h3>
+  <h3 class="item-grid__title type--h1">{{title}}</h3>
+  {% if modifiers.subtitle %}
+  <p class="item-grid__subtitle type--large">{{modifiers.subtitle}}</p>
+  {% endif %}
   <div class="item-grid__content">{% for item in items %} {% if modifiers.variant == 'icon' %} {{iconCard(item, defaultText)}} {% elif modifiers.variant == 'subnav' %} {% if item.icon and item.desc %} {{cardSubnav(item, locale)}} {% endif %} {% elif modifiers.variant == 'community' %} {{externalCommunity(item.logo, item.cta)}} {% else %} {{ card(item) }} {% endif %} {% endfor %}</div>
```

Comment:
This also looks different than some of the other changes at this point that I've seen; is there a canonical one I should look at?

Reply:
Apologies for that - there were multiple minor changes that ultimately should have warranted a PR of its own. Once pieced together, [this is the desired outcome](https://gist.github.com/nickalcantara/51ef2e70d9f1e6edfca23e3e38a03081), though there would also be additions for inner item components as needed.